ReasonVienna/reason-conf-2019
# Add interactive (Cookie notice)

How to test (in Chrome) after Accept:
- Open Devtools >> Application
- Select Cookies tab in sidebar
- Delete the value `gdpr_accept`
- Refresh page (now you should be able to see the message again)
